src/background/controller/wallet.ts

```diff
@@ -164,6 +165,103 @@ export class WalletController extends BaseController {
     return feeRatio.toString();
   };
 
+  sendERC20 = async ({
```

Comment:
```suggestion
  sendToken = async ({
```
Prefer sendToken since this function can send both ERC20 and native token